frontend/app/routes/$lang/_public/status/$id/result.tsx

```diff
@@ -0,0 +1,72 @@
+import { json } from '@remix-run/node';
+import type { LoaderFunctionArgs } from '@remix-run/node';
+import { useLoaderData, useParams } from '@remix-run/react';
+
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { useTranslation } from 'react-i18next';
+
+import pageIds from '../../../page-ids.json';
+import { ButtonLink } from '~/components/buttons';
+import { ClientFriendlyStatusMarkdown } from '~/components/client-friendly-status-markdown';
+import { ContextualAlert } from '~/components/contextual-alert';
+import { InlineLink } from '~/components/inline-link';
+import { useFeature } from '~/root';
+import { loadStatusState } from '~/route-helpers/status-route-helpers.server';
+import { featureEnabled, getEnv } from '~/utils/env-utils.server';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT } from '~/utils/locale-utils.server';
+import type { RouteHandleData } from '~/utils/route-utils';
+
+export const handle = {
+  i18nNamespaces: getTypedI18nNamespaces('status', 'gcweb'),
+  pageIdentifier: pageIds.public.status.myself,
+  pageTitleI18nKey: 'status:myself.page-title',
```

Comment:
I feel all these `status:myself.*` i18n keys should be refactored into `status:result.*` (in a future PR).

Reply:
Moved to new status lang section "result"